### ISSUE TITLE
Tests hangs if page redirect was cancelled

### ISSUE THREAD

**Issue by AlexanderMoskovkin** [DevExpress/testcafe#664]
The following code on the page after an action executed leads to the test hanging:

```js
// we are on the page http://example.com/page
window.location = 'http://other.domain.com';
window.setTimeout(function () {
    window.location = 'http://example.com/page#hash';
}, 200);
```

**Comment by inikulin** [comment]
@AlexanderMoskovkin Do I understand right that it was fixed by #759?

**Comment by AlexanderMoskovkin** [comment]
it was fixed by #759
